SETUP.md

```diff
@@ -104,102 +101,86 @@ External contributors can supply alternate placeholder values for secrets normal
 
 ### macOS
 
-These steps are for Apple devices running **macOS Monterey and Ventura**, including those running on [Apple Silicon (M1|M2)](https://en.wikipedia.org/wiki/Apple_silicon#M_series). 
+Setup steps for Apple devices running **macOS Ventura and Sonoma**, including those running on [Apple Silicon](https://en.wikipedia.org/wiki/Apple_silicon#M_series) and Intel x86:
```

Comment:
Include last two current macOS versions, explicitly mention that these instructions should (still) work on an x86 mac.